src/main/java/uk/gov/justice/hmpps/prison/service/digitalwarrant/DigitalWarrantService.java

```diff
@@ -204,26 +206,29 @@ public Integer createOffenderSentence(Long bookingId, Sentence sentence) {
 
         offenderSentenceChargeRepository.save(offenderSentenceCharge);
 
-        var movementReason = movementReasonReferenceCodeRepository.findById(MovementReason.SENTENCING).orElseThrow(EntityNotFoundException.withIdAndClass(MovementReason.SENTENCING.getCode(), MovementReason.class));
-        var eventStatus = eventStatusReferenceCodeRepository.findById(EventStatus.COMPLETED).orElseThrow(EntityNotFoundException.withIdAndClass(EventStatus.COMPLETED.getCode(), EventStatus.class));
-        var courtEvent = CourtEvent.builder()
-            .offenderBooking(booking)
-            .courtEventType(movementReason)
-            .eventStatus(eventStatus)
-            .startTime(sentence.getSentenceDate().atTime(10, 0))
-            .eventDate(sentence.getSentenceDate())
-            .courtLocation(courtCase.getAgencyLocation())
-            .offenderCourtCase(courtCase)
-            .build();
-
-        courtEventRepository.save(courtEvent);
-
-        var courtEventCharge = CourtEventCharge.builder()
-            .offenderCharge(offenderCharge)
-            .courtEvent(courtEvent)
-            .build();
-
-        courtEventChargeRepository.save(courtEventCharge);
+        /* TODO Removing this for now. It's unnecessary to add another event here. */
```

Comment:
Is this code definitely being uncommented later?

Reply:
Yea I've got a ticket tracking these changes, I'll make sure I add this comment so that we clean it up later.